dev-haskell/hmatrix/Manifest

```diff
@@ -1 +1,2 @@
 DIST hmatrix-0.20.0.0.tar.gz 102180 BLAKE2B fbbe38a89813fc4ca8fdfc21a46cd5ae36a5e0fe05f742aceb2febdfbc6a2fcd1653bc5887859d6d3ff49841a1265253765cee4679c8f6338978a808a7bec18d SHA512 1845bd5a88d589a7b7543de3543747a3de642ca0801e1483e5022dd34cffc2229b478d6f91fde5fe0d525365c8644ce4a67da9bc32ea2872845531f339947d7c
```

Comment:
You might as well delete the old version since it's no longer building.

Reply:
removed old version and patches, cleaned up the manifest as requested.

Reply:
Thanks, although it looks like you need to use `git mergetool` or manually get rid of everything execpt line 4